app/src/main/java/me/ccrama/redditslide/Activities/SettingsHandling.java

```diff
@@ -36,11 +38,6 @@
     @Override
     public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
         switch (buttonView.getId()) {
-            case R.id.web:
```

Comment:
This case confused me what it was doing since the rest of the browser_type menu options were not in here. Can you explain it? Removing it seemed to not affect anything.